.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Does this file get created by `husky install`?

Reply:
Right, This is generated by husky when using `npx husky --init`. This would only be one time, If you want to add more hooks you most put in `./husky`